love/src/components/MainTel/CameraHexapod/CameraHexapod.jsx

```diff
@@ -0,0 +1,316 @@
+      { hexapodApplication: 'Compensation Offset[um, deg]', ...defaultValues },
+    ];
+
+    for (let i = 0; i < this.props.hexapodApplicationDemand.length; i++) {
+      const axis = ['x', 'y', 'z', 'u', 'v', 'w'];
+      dataHexapod[0][axis[i]] = this.props.hexapodApplicationDemand[i];
+    }
+
+    for (let i = 0; i < this.props.hexapodApplicationPosition.length; i++) {
+      const axis = ['x', 'y', 'z', 'u', 'v', 'w'];
+      dataHexapod[1][axis[i]] = this.props.hexapodApplicationPosition[i];
+    }
+
+    dataHexapod[2].x = this.props.hexapodCompensationOffsetX;
+    dataHexapod[2].y = this.props.hexapodCompensationOffsetY;
+    dataHexapod[2].z = this.props.hexapodCompensationOffsetZ;
+    dataHexapod[2].u = this.props.hexapodCompensationOffsetU;
+    dataHexapod[2].v = this.props.hexapodCompensationOffsetV;
+    dataHexapod[2].w = this.props.hexapodCompensationOffsetZ;
+
+    const compensation = this.props.hexapodCompensationMode;
+    const newDataHexapod = dataHexapod.filter((val, index) => index !== 2 || (index === 2 && compensation === true));
+
+    // Strut Length Table
+    const defaultsValues = { 1: 0, 2: 0, 3: 0, 4: 0, 5: 0, 6: 0 };
+    const dataStrut = [{ strutLength: 'Actual [um]', ...defaultsValues }];
+
+    for (let i = 0; i < this.props.hexapodActuatorsCalibrated.length; i++) {
+      const axis = ['1', '2', '3', '4', '5', '6'];
+      dataStrut[0][axis[i]] = this.props.hexapodActuatorsCalibrated[i];
+    }
+
+    // Hexapod status and Readiness summary
+    const hexapodStatus = CSCDetail.states[this.props.hexapodSummaryState];
+    const compensationStatus = hexapodCompensationModeStateMap[this.props.hexapodCompensationMode];
+    const controllerState = hexapodControllerStateMap[this.props.hexapodControllerState];
+    const interlockState = hexapodInterlockStateMap[this.props.hexapodInterlock];
+
+    const commandableByDDS = hexapodCommandableByDDSStateMap[this.props.hexapodCommandableByDDS];
+
+    // controllerState
+    let controllerSubstate = '';
+    if (controllerState === 'Offline') {
+      controllerSubstate = hexapodControllerStateOfflineSubStateMap[this.props.hexapodControllerStateOfflineSubstate];
+    }
+    if (controllerState === 'Enabled') {
+      controllerSubstate = hexapodControllerStateEnabledSubstateMap[this.props.hexapodConstrollerStateEnabledSubstate];
+    } else {
+      controllerSubstate = 'Offline';
+    }
+
+    const inPosition = hexapodMTInPositionStateMap[this.props.hexapodInPosition];
+    const connected = hexapodConnectedStateMap[this.props.hexapodConnected];
+
+    return (
+      <div>
+        <div className={styles.hexapodContainer}>
+          <SummaryPanel className={styles.summaryPanel}>
+            <Title wide>Hexapod Status</Title>
+            <Label>Hexapod Status</Label>
+            <Value>
+              <StatusText status={hexapodStatusStatetoStyle[hexapodStatus.name]}>{hexapodStatus.name}</StatusText>
```

Comment:
Remember that for `hexapodStatus` we are using a dictionary to map states from `CSCDetail`. The value we return at line 240 is an object with different properties: `name` and `class` are the relevant ones. Thus you don't have to give a `status` value to the `StatusText` component (as you already have an style object inside the `class` param), you have to set a `div` with `className={hexapodStatus.class}`

Reply:
Also remember to give the required style to be a rounded label (as now you won't be using `StatusText` component)